easybuild/easyconfigs/k/KronaTools/KronaTools-2.8-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,40 @@
+# Contribution from the Crick HPC team
```

Comment:
See  #12871 for suggested changes to the existing KronaTools ecs. This should be changed in the same way.